src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -15,7 +15,7 @@
 namespace System.Diagnostics.Tests
 {
     //Complex types are not supported on EventSource for .NET 4.5
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework45)]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Net45 | TargetFrameworkMonikers.Net451 | TargetFrameworkMonikers.Net452)]
```

Comment:
Yes, but when I did the PR on build tools to update the TargetFrameworkMonikers, Wes told me to leave the versioned ones in case we want to go back to versioned tests. So if the test is skipped on a previous version and passes against netfx I was thinking of leaving it as it is. What do you think?